src/newshop.cpp

```diff
@@ -1317,8 +1319,8 @@ void shop_list(char *arg, struct char_data *ch, struct char_data *keeper, vnum_t
 
 
   if (shop_table[shop_nr].flags.IsSet(SHOP_DOCTOR)) {
-    strlcpy(buf, " **   Availability     Item                              Rating  Ess/Index     Price\r\n"
-                 "------------------------------------------------------------------------------------\r\n", sizeof(buf));
+    strlcpy(buf, " **   Availability     Item                                                           Rating  Ess/Index     Price\r\n"
+                 "-----------------------------------------------------------------------------------------------------------------\r\n", sizeof(buf));
```

Comment:
The longer we make hardcoded strings like this, the more likely someone will have wrapping issues. Is it necessary to add this much spacing, or can we get away with making this shorter and condensing information elsewhere to fit?

Reply:
This is using the space that the currently longest string we have takes. And the width problem is already there, meaning the string output of the actual items are already that long and they just mess up the table. So basically this is fixing the padding so the tables print fine. It is 1 character longer than what the current longest string requires which is this: `a pair of Saeder-Krupp "Panzerfaust" obvious cyberarms -beta` (It's not even a huge string heh it's that we're printing a lot of columns on shop tables).

Now can we condense stuff? We probably can. We can shorten the special order output be removing the "Sp." part, we can change "Cyber" to "Cyb", little stuff like that to save 2 characters here, 2 characters there. The real question though is, is it worth it? We're at 2022 and there are really no 80 char wide terminals anymore. Several command outputs are already beyond the `LINE_LENGTH-1` because simple they don't fit in that no matter what. Monitors and all mud clients, including plain telnet, do not have that issue anymore, it's not the 80s, heh.
I have not checked for it but since this is the Kavir protocol options code, NAWS is probably enabled as well (unless it was never implemented entirely here but I've seen the nego code talking abou t it) which means all modern (as in more than a decade old :P ) clients can negotiate their wrapping width too. Even BlowTorch on android does not have an 80 char width limitation. MUD output on my monitor, and I'm sure on yours too, takes like 1/4th of available space with the rest being blank. Is it really worth beating this dead horse? :P

Reply:
Nah, we can leave it be then and see how it goes over.